<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QP76K1J" failures="3" tests="19" name="org.example.testRunner.runners" time="162.488" errors="1" timestamp="2024-02-15T19:30:17 EET" skipped="0">
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="5.966"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="8.194">
    <error message="Index 1 out of bounds for length 1" type="java.lang.IndexOutOfBoundsException">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:385)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at org.example.pages.P03_homePage.handleSecondPage(P03_homePage.java:241)
at org.example.stepDefinitions.D07_FollowUsStepDef.user_redirect_to_RSS_page_url(D07_FollowUsStepDef.java:50)
at ✽.user redirect to RSS page URL(file:///E:/Automation/nopcommerce-testing-main/src/main/resources/Features/F07_FollowUs.feature:18)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="7.250"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="7.981">
    <failure message="Unexpected URL: https://demo.nopcommerce.com/" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unexpected URL: https://demo.nopcommerce.com/
at org.testng.Assert.fail(Assert.java:111)
at org.example.pages.P03_homePage.validateUrls(P03_homePage.java:228)
at org.example.stepDefinitions.D06_HomeSlidersStepDef.validate_on_url_is_right(D06_HomeSlidersStepDef.java:23)
at ✽.validate the URL is right(file:///E:/Automation/nopcommerce-testing-main/src/main/resources/Features/F06_HomeSliders.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="5.990"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="5.944"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="28.330"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="7.080"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="6.857"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="5.405"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="7.050"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="5.985">
    <failure message="The following asserts failed:
	Search result verification failed for result #2 expected [true] but found [false],
	Search result verification failed for result #6 expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Search result verification failed for result #2 expected [true] but found [false],
Search result verification failed for result #6 expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at org.example.pages.P03_homePage.searchResultsForSearchProduct(P03_homePage.java:162)
at org.example.stepDefinitions.D04_SearchStepDef.search_results_for_product_are_displayed(D04_SearchStepDef.java:23)
at ✽.search results for "book" are displayed(file:///E:/Automation/nopcommerce-testing-main/src/main/resources/Features/F04_Search.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="6.277"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="6.881"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="10.423"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="10.684"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="5.888"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="13.006"/>
  <testcase classname="org.example.testRunner.runners" name="runScenario" time="7.297">
    <failure message="Unexpected URL: https://demo.nopcommerce.com/" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unexpected URL: https://demo.nopcommerce.com/
at org.testng.Assert.fail(Assert.java:111)
at org.example.pages.P03_homePage.validateUrls(P03_homePage.java:228)
at org.example.stepDefinitions.D06_HomeSlidersStepDef.validate_on_url(D06_HomeSlidersStepDef.java:37)
at ✽.validate the URL(file:///E:/Automation/nopcommerce-testing-main/src/main/resources/Features/F06_HomeSliders.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- org.example.testRunner.runners -->
